### DISCUSSION TITLE
Pandas works on Dataset but Vaex does not

### DISCUSSION THREAD

**Discussion by abhatia1** [vaexio/vaex#2015] [Q&A]
I am trying to switch over to Vaex and I am having some trouble. The dataset is about 3.5 GB, I am able to read file in with from_csv, but any operation instantly kills the kerneI (even df.describe()). I tried using chunk size and that also killed the kernel instantly. However, I tried importing the dataset with pandas and it worked fine and the operations also worked (df.describe, df.column.unique()). Can you help me understand what is going on here and why Vaex kills the kernel but pandas works? Thank you!

**Comment by JovanVeljanoski** [comment]
It would be helpful to provide more information, some ideas:
-  Is vaex successfully installed
  - which version / OS / environment manager
  - Can you do `df = vaex.example()` and do operations on that
- Is there something special about your csv file (i.e. non standard dtypes etc..)

We need something to reproduce the issue, otherwise we can't help..

**Reply by abhatia1** [reply]
Hi there, thanks for the ideas:

- Vaex is successfully installed here are the specs:
- {'vaex': '4.9.0',
 'vaex-core': '4.9.0',
 'vaex-viz': '0.5.1',
 'vaex-hdf5': '0.12.1',
 'vaex-server': '0.8.1',
 'vaex-astro': '0.9.1',
 'vaex-jupyter': '0.7.0',
 'vaex-ml': '0.17.0'}

However, when I do vaex.example() I get this:
URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1076)>

Using Python 3.7 on Mac OS Monterey, nothing special about the csv just a lot of rows.

Let me know if this helps or if you need any other info, thank you for helping me get started!

**Comment by maartenbreddels** [comment]
This is probably not vaex related, I suggest you create a new environment (conda or venv) to make sure all your packages are up to date.
